apps/ledger-live-desktop/src/config/urls.ts

```diff
@@ -138,11 +138,7 @@ export const urls = {
     "https://support.ledger.com/hc/en-us/articles/115005165269?utm_source=ledger_live_desktop&utm_medium=self_referral&utm_content=error",
   troubleshootingCrash:
     "https://support.ledger.com/hc/en-us/articles/360012598060?utm_source=ledger_live_desktop&utm_medium=self_referral&utm_content=error",
-  appSupport: {
-    "Cardano ADA":
-      "https://support.ledger.com/hc/en-us/articles/360020095874-Cardano-ADA-?docs=true",
-    default: "https://support.ledger.com/hc/en-us/sections/4404369637521-Crypto-assets?docs=true",
-  },
+  appSupport: "https://support.ledger.com/hc/en-us/sections/4404369637521-Crypto-assets?docs=true",
```

Comment:
do we need the `utm_source=ledger_live_desktop` GET params for this link ? Or we don't care ? They were added for LLM

Reply:
If we do, it can be another ticket as far as I'm concerned.